Nginx-Certbot.md

```diff
@@ -0,0 +1,181 @@
+    # this line is for the domain name you chose
+    # make sure the domain name matches the one you bought!
+    server_name phry.me lol.phry.me;
+
+    # these lines will help protect users from accessing raw files in your repository folder
+    location / {
+        try_files $uri $uri/ =404;
+    }
+}
+```
+
+Save and close the configuration file.
+
+Next, we need to set up a symbolic link to this raw file in the 
+`/etc/nginx/sites-enabled` directory, so that `nginx` serves the 
+content.
+
+Run these commands:
+
+```
+$ cd /etc/nginx/sites-enabled
+$ ln -s ../sites-available/phry.me .
+```
+
+_Note: don't forget to replace `phry.me` with the filename you chose 
+when creating the configuration file in `/etc/nginx/sites-available`._
+
+Now, we need to restart the `nginx` process on the machine.
+
+```
+$ systemctl restart nginx.service
+```
+
+If you do not see an error code on the last command, then `nginx` 
+successfully restarted and should be serving the content from the 
+cloned repository.
+
+_If you do see an error message, seek help from a hackathon mentor!_
+
+## Step 4: Enable HTTPS Traffic to Your Domain with CertBot
+
+[CertBot](https://certbot.eff.org/) is a tool created by the 
+[Electronic Frontier Foundation](https://www.eff.org/) to deploy 
+SSL/TLS encryption certificates on your machine so that your web 
+traffic can be served over HTTPS. This helps ensure your website and 
+the user accessing your site remain secure.
+
+To get started, install the necessary packages by running these 
+commands:
+
+```
+$ sudo apt-get update && apt-get upgrade
+$ sudo apt-get install software-properties-common
+$ sudo add-apt-repository ppa:certbot/certbot
+$ sudo apt-get update
+$ sudo apt-get install python-certbot-nginx
+```
+
+Now that the CertBot client is installed, generate a certificate for 
+your site by running the following command:
+
+```
+$ certbot run
+```
+
+You will be prompted for the following information:
+- **Enter an email address:** you will receive emails at this address 
+  when the certification is due for renewal, or if any important 
+  security notices need to be shared with you.
+- **Agree to Terms of Service for Let's Encrypt:** Enter `A` to 
+  continue.
+- **Share contact information with the EFF:** this is completely 
+  optional, but EFF is a great organization and it's up to you as to 
+  whether you want to receive correspondence from them.
+- **Choose domains to activate HTTPS for:** CertBot will 
+  automatically detect domains that `nginx` is serving content for. 
+  It is advisable to add HTTPS certificates for all domains.
+  You can:
+  - Select multiple domains individually by numbered listings 
+    on the console prompt
+  - Leave the input blank to add certificates for all domains listed
+  - Enter `c` to cancel the certification process
+- **Choose whether to redirect HTTP traffic to HTTPS:** selecting 
+  option 2 will redirect all HTTP traffic to HTTPS; this is the best 
```

Comment:
might want to mention that this will update the nginx configuration files

Reply:
Addressed in 7c1a9d051f9c55fa86e76013b274a40a644cea45